Pod/Classes/GDGEntityManager.h

```diff
@@ -38,6 +38,8 @@
 
 - (void)hasMany:(NSString *)relationName config:(void (^)(GDGHasManyRelation *))tap;
 
+- (void)hasManyThrough:(NSString *)relationName config:(void (^)(GDGHasManyThroughRelation *))tap;
```

Comment:
Not saying that this way is wrong, or anything, but I was imagining something like `-hasMany:through:config:`